.changeset/polite-bottles-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-zod-client": minor
```

Comment:
it was 'major' from an auto generated changeset, but I'm not sure if this modification qualifies as a major change, so I manually adjusted it to a minor one